src/OWASPZAPDotNetAPI/NugetPackaging/OWASPZAPDotNetAPI.nuspec

```diff
@@ -2,17 +2,20 @@
 <package >
   <metadata>
     <id>OWASPZAPDotNetAPI</id>
-    <version>1.0.0</version>
+    <version>2.6.0</version>

```

Comment:
makes sense. 

I prefer to keep it this way. However, let's say there is a bug in the code, releasing it as 2.6.1 makes sense to me. There is also release notes that can be updated acoordingly in the nuget package manager.

Having said that, if there is a convention followed for the Java Client API, it is just easy to follow the same convention throught all API clients, including the Dot Net API.